app/test_main.py

```diff
@@ -1,3 +1,63 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        pytest.param(
+            0,
+            0,
+            [0, 0],
+            id="cat and dog ages equal zero"
+        ),
+        pytest.param(
+            14,
+            14,
+            [0, 0],
+            id="cat and dog ages less then human"
+        ),
+        pytest.param(
+            15,
+            15,
+            [1, 1],
+            id="15 cat/dog years should convert into 1 human age."
+        ),
+        pytest.param(
+            23,
+            23,
+            [1, 1],
```

Comment:
This test case seems to repeat the logic of the previous one (15 cat/dog years). Ensure that each test case checks a unique scenario to adhere to the 'Don't Repeat Yourself' principle.